src/data/cart.data.js

```diff
@@ -0,0 +1,31 @@
+import { cartImgArr, sizes } from './cartImg.data'
+
+export const cartItmArr = [
+    {
+        id: 1,
+        name: 'air max pre day',
+        company: 'nike',
+        img: {
+            srcSet: cartImgArr[0].srcSet,
+            alt: cartImgArr[0].alt,
+            sizes: sizes
+        },
+        size: 9,
+        quantity: 1,
+        price: 11789,
+        offerPrice: 10000
+    },
```

Comment:
suggestion - you APIs for data fetching

Reply:
I'll create a seperate pr for APIs, so that is why i'm using this dummy data.